crates/lowering/tests/snapshots/lowering__module_const.snap

```diff
@@ -0,0 +1,21 @@
+---
+source: crates/lowering/tests/lowering.rs
+expression: lowered_code
+
+---
+const SOME_THING: u256 = 1 + 2
```

Comment:
can we drop these definitions from the AST?

Reply:
Makes sense :+1: